src/cta-button/editor.scss

```diff
@@ -0,0 +1,97 @@
+/**
+ * #.# Editor Styles
+ *
+ * CSS for just Backend enqueued after style.scss
+ * which makes it higher in priority.
+ */
+
+.bulma-blocks-editor-link-card {
+
+  padding: 0.875rem;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: #000;
+  background: #f3f3f3;
+  margin-bottom: 1.75rem;
+
+  .content {
+    width: 100%;
+    max-width: 30rem;
+
+    span {
+      display: flex;
+      margin-bottom: 1rem;
+    }
+
+    &:first-child {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+    }
+    .field{
+      .control{
+        input{
+          background-color:#fff;
+        }
+      }
+    }
+  }
+
+  &__container {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background: #f3f3f3;
+    color: black;
+    font-family: sans-serif;
+
+    p {
+      text-align: center;
+    }
+  }
+
+  &__heading {
+    font-size: 0.9rem;
+    margin-bottom: 0.75rem;
+    display: flex;
+    align-items: center;
+
+    .dashicons {
+      margin-right: 0.75rem;
+    }
+  }
+
+  &__description {
+    width: 80%;
+    font-size: 0.85rem !important;
+    margin: 0 !important;
+    margin-bottom: 0.75rem !important;
+
+    &--error {
+      color: red !important;
+    }
+  }
+
+  &__button {
+    border: #00c4a7 1px solid;
```

Comment:
noticed a lot of repeated colors in this repo, would it be possible to have a _var.scss where you store these colors in variables